### ISSUE TITLE
EIPIP Meeting 100

### ISSUE THREAD

**Issue by poojaranjan** [ethcatherders/EIPIP#312]
# Date and Time

Feb 14, 2024 at 17:30 UTC 

# Location

Zoom: TBA in the Discord #eip-editing channel

YouTube Recording: [EIPIP Meetings](https://www.youtube.com/playlist?list=PL4cwHXAawZxpLrRIkDlBjDUUrGgF91pQw)

# Agenda

## 1. EIP Process Standardization

- [x] Adding "Errata" section for Final EIPs.
  - Open a pull request to modify EIP-1 and `eip-template.md`

- [x] Working Group [Charter](https://hackmd.io/@SamWilsn/rk2CG6qqp/edit)

## 2. Discuss Open Issues/PRs, and other topics

- [x] Merge https://github.com/ethereum/ERCs/pull/213 ? [Ref](https://github.com/ethcatherders/EIPIP/issues/310#issuecomment-1931872291)

### Call for Input

Call For Input | Status | Result | Comments
-- | -- | -- | --
https://github.com/ethcatherders/EIPIP/issues/306 | Open |  | Deadline Feb 07, 2024
https://github.com/ethcatherders/EIPIP/issues/308 | Open  | | Deadline Feb 08, 2024

## 3. Other discussions continued or updates from past meetings
- [ ] RIP process documentation
- [ ] Web Page Rendering to include EIPs, ERCs, RIPs

## 4. EIPs Insight - Monthly EIPs status reporting.
- [EIPsInsight.com](https://eipsinsight.com/insight/2024/2)
- [Hackmd](https://hackmd.io/@poojaranjan/EthereumImprovementProposalsInsight)

## 5. [EIP Editing Office Hour](https://hackmd.io/@poojaranjan/EIPEditingOfficeHour)
- Agenda [Meeting 32](https://github.com/ethcatherders/EIPIP/issues/311) Video [31](https://www.youtube.com/watch?v=Z9GmY1hJ2Ao)

## 6. Review action items from earlier meetings
- [Summary meeting 99](https://github.com/ethcatherders/EIPIP/issues/310#issuecomment-1920381874)

## Next Meeting date & time 

Feb 28, 2024 at 17:30 UTC

**Comment by gcolvin** [comment]
We have long needed an Errata section for Final EIPs.  Probably at the end.  Either we change the original and call out the change in the Errata section, or call out the error in the Errata section.  But leaving mistakes.  Either works, and either can be supported by Github.

**Comment by SamWilsn** [comment]
If you'd like a formal decision on adding an errata section, I'd suggest opening a pull request to modify EIP-1 and `eip-template.md`, and then opening a Call for Input to collect input from all the Editors. Don't forget to post the call for input in Discord.

I normally use this template:

```markdown
## Call for Input

<table>
<tr>
<th>Decision</th>
<td>

<!-- Pose a question that Editors can agree or disagree with -->

</td>
</tr>
<tr>
<th>If Affirmed</th>
<td>

<!-- Describe what happens if the rough consensus is to answer the question in the affirmative -->

</td>
</tr>
<tr>
<th>If Rejected</th>
<td>

<!-- Describe what happens if the rough consensus is to reject the question -->

</td>
</tr>
<tr>
<th>Method</th>
<td>Rough Consensus</td>
</tr>
<tr>
<th>Deadline</th>
<td>

<!-- 30 days from when the CFI is created -->

</td>
</tr>
</table>

# Background

<!-- Include any relevant context for EIP Editors here -->
```

**Comment by gcolvin** [comment]
I think we need to discuss it first, given there are at least two ways to handle them.

**Comment by poojaranjan** [comment]
Added for discussion @SamWilsn @gcolvin !

**Comment by poojaranjan** [comment]
## Summary

### 1. EIP Process Standardization
 #### Adding "Errata" section for Final EIPs.
- @gcolvin advocated for the need for "Errata" for [PR](https://github.com/ethereum/ERCs/pull/184)
- @xinbenlv supports adding "Errata" section to the proposal.
- @g11tech is not in favor of allowing changes to a Final EIP. He suggested to just move away from `authors` to `maintainers`, people can comment on authorship in discussion to link.
- Sam also mentioned the proposal for [EIP Versioning](https://ethereum-magicians.org/t/add-eip-versioning-scheme-for-eips/17295) by testing team. It may help.
- @SamWilsn as the "keeper of consensus" highlighted that [Call for Input: Adding Additional Author to ERC-5169](https://github.com/ethcatherders/EIPIP/issues/308) opened to collect editors' feedback was open for over a month. And the decision is "No new author is added." 
- Unless there is any new argument by the present author, the decision will remain the same.
- Any new author/proposer for an Ethereum standard must be sure that the proposal will not be updated including the list of authors,  once moved to `Final` status.
- Long discussion, can be followed on the recording. 

 #### Working Group [Charter](https://hackmd.io/@SamWilsn/rk2CG6qqp/edit)
- @SamWilsn shared the link to collect feedback and is willing to present it to the editors' group in the upcoming meeting. 

### 2. Discuss Open Issues/PRs, and other topics
 #### https://github.com/ethereum/ERCs/pull/213
- @SamWilsn & @g11tech are okay to merge it. 
- Sam will merge the PR asap.

#### Call for Input

##### https://github.com/ethcatherders/EIPIP/issues/306	
- It is for back references for a later EIP to an earlier one.
- It would look like ERC 223 would have a header in it  - `Supersedes: 20` and on ERC-20 page there will be a list of all of the things of ERC-223 claiming to supersede ERC-20. This will improve navigation between proposals that claim to supersede. 
- @SamWilsn is in favor of the proposal.
- @g11tech is not in favor. However, as long as clutter can be managed, he'll be fine with it. 
- @gcolvin hasn't added input on the link
- Requesting editors to add comments asap. 

### 3. Other discussions continued or updates from past meetings

#### RIP process documentation
- No new info is collected from the RIP team. However, any new documentation/ explanation for the process will be highly appreciated. 
#### Web Page Rendering to include EIPs, ERCs, RIPs
- As of date, eips.ethereum.org includes EIPs and RIPs data but does not render RIPs. @poojaranjan will inform the group if there is any success in rendering RIPs.
- In parallel, @SamWilsn is working on page rendering based on the working group. The output will contain RIP data as well. 

### 4. EIPs Insight - Monthly EIPs status reporting.
[EIPsInsight.com](https://eipsinsight.com/insight/2024/2)
[Hackmd](https://hackmd.io/@poojaranjan/EthereumImprovementProposalsInsight)

### Next Meeting date & time
Feb 28, 2024 at 17:30 UTC

**Comment by poojaranjan** [comment]
Closing in favor of #316
